### DISCUSSION TITLE
command 'github.copilot.interactiveSession.feedback' not found

### DISCUSSION THREAD

**Discussion by alko89** [community/community#77195] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Recently noticed Copilot Chat is not available without requiring the insiders version of code, which I prefer because I use my package manager to update it (have to do manually with insiders :tired_face: ).

Anyway when trying to enable it I get the error: command 'github.copilot.interactiveSession.feedback' not found

I've tried logging out and back in with no effect.

I can see the chat prompt but am unable to send anything

![image](https://github.com/community/community/assets/2822030/b30a3afd-22f0-460f-8925-9df265a3040a)

**Comment by Akash1134** [comment]
Hi @alko89 ,

Have you tried signing out with the below process?

- In the bottom left corner of the Visual Studio Code window, click the Accounts icon, hover over your GitHub username, and click the Sign out button.
- To reload Visual Studio Code, press F1 to open the command palette, and select Developer: Reload Window.
- After Visual Studio Code reloads, sign back in to your GitHub account.

Thanks!

**Reply by alko89** [reply]
Yep, I found this page and tried it, but didn't seem to help.

https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-issues-with-github-copilot-chat?tool=vscode#troubleshooting-authentication-issues-in-visual-studio-code

**Comment by JSantora** [comment]
I have exactly same issue with the chat. Followed your bullet points and also no luck. 
- in outputs for GitHub copilot Chat, there is nothing
- copilot autocomplete is working for me
- assigned to single org

**Comment by digitarald** [comment]
@alko89 as you are using stable VS Code, can you confirm that you are using Copilot Chat's release version as well?

If it persists, please file the issue with the error logs at https://github.com/microsoft/vscode-copilot-release

**Reply by alko89** [reply]
@digitarald, this is the version that seem to have install itself automatically

![image](https://github.com/community/community/assets/2822030/f6a23030-b79f-4f73-a19a-1a4de2ac6d44)

 I've checked the logs but it seems nothing related to chat is there at all, only line I see is

```
2023-12-02 13:07:10.177 [info] ExtensionService#_doActivateExtension GitHub.copilot-chat, startup: false, activationEvent: 'onInteractiveSession:copilot'
```

Do I have to enable debug logs for the extension somehow?

**Reply by JSantora** [reply]
Hey hey, I have figured out what my issue has been. 
I had a workspace with multiple repositories, when opening the workspace copilot chat did not work, when outside of the workspace, it did. Very odd. I am thinking if I have some repos which copilot is disabled for or I am no longer able to access.

**Comment by sdcanny** [comment]
@alko89 Any luck with this? I've got the same issue.

**Reply by digitarald** [reply]
@sdcanny can you open an issue with your logs in https://github.com/microsoft/vscode-copilot-release/issues

**Reply by bydmiller** [reply]
To fix the problem, follow these steps:
1. Disable the Copilot chat extension.
2. Restart Visual Studio Code.
3. Enable the extensions.
Hope this work.

**Comment by AlbertRoh** [comment]
Was this problem resolved? While using VS Code remotely in a High-Performance Computing environment, I encountered the same issue. I have tried the steps recommended in the previous comments, but they did not work for me. Could anyone please share the approaches you took if you've encountered a similar situation? Thank you!

**Reply by Maurikzio** [reply]
Hi, @AlbertRoh I had the same issue, and also following the steps in the previous comments didn't work for me. It started working for me reloading Visual Studio Code when the extension itself asks to do so. So, check the extension and see if it is asking to reload if so a button in the extension page/docs will show up and click on it.
I hope it helps, good luck 🤞 .

**Comment by felix-kocht** [comment]
I have the same issue and all of the proposed steps have not solved it. when clicking the person with speech bubble icon up top i get this error message: command 'github.copilot.interactiveSession.feedback' not found. Github copilot in the code itself works perfectly. Any more ideas?

**Reply by bbm16** [reply]
I just made it work on latest version Copilot (v1.162.0) and Copilot Chat (v0.12.2).

I had the same issue, try to signout / signin to your github account on vscode and then reload the chat extension, hope it helps.

**Reply by davecheney** [reply]
@bbm16 you need to upgrade, please see https://github.com/orgs/community/discussions/101438 for details.

**Comment by skrabe** [comment]
Same problem, if I try clicking the icon at the top I get "github.copilot.interactiveSession.feedback". Tried reloading the extension, VSCode window, logging out/in with all possible combinations, I just can't get it to work. Autocomplete works as intended though, so that's very weird.

**Comment by mawenge** [comment]
Hi guys, this worked for me, hope could help you:

1. switch copilot chat plugin version to pre-release version, now it is:v0.12.2023120701
2. sign out github account
3. restart vscode
4. sign in github accout
then the copilot chat works

**Reply by davecheney** [reply, marked as answer]
_DO NOT USE THE OLD VERSION OF COPILOT CHAT_

_please_

1. switch from -insiders to regular vscode, -insiders is no longer required for chat functionality
2. install the copilot extension from the vscode marketplace, keep hitting update until vscode is happy.

**Reply by skrabe** [reply]
@mawenge this one worked, out of all the possible combinations ive tried with reload/uninstall/switch to (pre)release/logout, this one seems the only one to actually work.

**Comment by mcmoe** [comment]
Signing out and back in worked for me.

**Comment by davecheney** [comment]
Closing as duplicate of https://github.com/orgs/community/discussions/110196. C'mon folks, y'all should upgrade, it's better.
